doc/topics/installation/debian.rst

```diff
@@ -181,7 +180,7 @@ To install fresh release of Salt minion on Jessie:
    .. code-block:: bash
 
        apt-get update
-       apt-get install python-zmq python-tornado/jessie-backports salt-common/stretch
```

Comment:
I think it's still better to go with `jessie-backports` on pure Debian. So, please make the case for _Debian_ and _Raspbian_ here, like:

**Debian**:
```
apt-get install python-zmq python-tornado/jessie-backports salt-common/stretch
```

**Raspbian**:
```
apt-get install python-zmq python-tornado/stretch salt-common/stretch
```